packages/wdio-allure-reporter/src/index.js

```diff
@@ -120,6 +124,20 @@ class AllureReporter extends WDIOReporter {
         currentTest.addLabel('thread', cid)
     }
 
+    getLabels(test){
+        var tags = test.tags
+        var labels = []
+        if (tags != null && tags != undefined){
+            tags.forEach(function (tag) {
+                var label = tag.name.replace(/[@]/, '').split('=')
+                if(label.length == 2){
+                    labels.push({ name: label[0], value: label[1] })
+                }
+            })
```

Comment:
Could you explain why did you remove line with `console.log`?

Reply:
Many irrelevant logs would have been displayed if tags other than "Allure" tags are present in feature file